devito/operator/profiling.py

```diff
@@ -367,21 +376,23 @@ class AdvisorProfiler(AdvancedProfiler):
     _ext_calls = [_api_resume, _api_pause]
 
     def __init__(self, name):
-        self.path = locate_intel_advisor()
-        if self.path is None:
-            self.initialized = False
+        if self._attempted_init:
```

Comment:
shouldn't this go _after_ the `super()` below?

Reply:
No, otherwise, we would change the class attributes again! And then we would have the same problem as why it was not working before. include dirs and libdirs will not be included